__tests__/implementation.test.js

```diff
@@ -113,4 +113,43 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("should respond with a status 200 and array of comments for the given article_id", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          const expectedProperties = [
+            "comment_id",
+            "votes",
+            "created_at",
+            "author",
+            "body",
+            "article_id",
+          ];
+
+          response.body.comments.forEach((comment) => {
```

Comment:
please rewrite this test. Just take the array of comments and use a single forEach on it to test that each comment has a the properties you expect. Then for the article_id test the actual_id as you know it needs to be 1.